lib/src/flutter/code_generation/generators/gherkin_suite_test_generator.dart

```diff
@@ -51,14 +51,14 @@ class _CustomGherkinIntegrationTestRunner extends GherkinIntegrationTestRunner {
   {{feature_functions}}
 }
 
-void executeTestSuite({
+Future executeTestSuite({
   required FlutterTestConfiguration configuration,
   required StartAppFn appMainFunction,
   Timeout scenarioExecutionTimeout = const Timeout(const Duration(minutes: 10)),
   AppLifecyclePumpHandlerFn? appLifecyclePumpHandler,
   LiveTestWidgetsFlutterBindingFramePolicy? framePolicy,
-}) {
-  _CustomGherkinIntegrationTestRunner(
+}) async {
+  return _CustomGherkinIntegrationTestRunner(
```

Comment:
```suggestion
}) =>
    _CustomGherkinIntegrationTestRunner(
```
while removing the end bracket `}`